control/statefbk.py

```diff
@@ -205,19 +248,19 @@ def place_varga(A, B, p, dtime=False, alpha=None):
             # the same as what slicot thinks are the eigs. So we need some
             # numerical breathing room. The following is pretty heuristic,
             # but does the trick
-            alpha = -2*abs(min(system_eigs.real))
+            alpha = -2 * abs(min(system_eigs.real))
     elif dtime and alpha < 0.0:
-        raise ValueError("Need alpha > 0 when DICO='D'")
-
+        raise ValueError("Need alpha > 0 when dico='D'.")
```

Comment:
This error message is confusing since `dico` is not the name of the parameter that is passed to the function (it is `dtime`).